react/Header/Navigation/Navigation.less

```diff
@@ -61,13 +61,15 @@
 }
 
 .newBadge {
-  margin-left: 5px;
   @media only screen and (max-width: @container-width) {
     position: absolute;
-    top: -4px;
-    right: (@gutter-width / 2);
+    right: 0;
+    top: -@touchableTextHeight / 2 + 2px;
     &.newBadge_isShort {
-      right: @gutter-width * 1.5;
+      right: 17%;
```

Comment:
Is this correct as a percentage? Seems an odd number thats all

Reply:
Yeah, it's more correct as a percentage, it's weird because it's custom and depends on the width of the tab and copy on the badge. So there can't be anything not odd